tests/test_input.py

```diff
@@ -0,0 +1,385 @@
+# Copyright 2023 Google LLC.
+# Copyright (c) Microsoft Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from anys import ANY_STR
+from test_helpers import execute_command, goto_url
+
+SCRIPT_HTML = 'data:text/html,' + """
```

Comment:
Define the script without `data:text/html`. Then, in the test function, add the `html` fixture and use it as `html(SCRIPT)`